sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/ChatThreadProperties.java

```diff
@@ -122,6 +130,46 @@ public ChatThreadProperties setCreatedBy(CommunicationIdentifier createdBy) {
         return this;
     }
 
+    /**
+     * Get the metadata property: Property bag of chat thread metadata key-value pairs.
+     *
+     * @return the metadata map.
+     */
+    public Map<String, String> getMetadata() {
+        return this.metadata;
+    }
+
+    /**
+     * Set the metadata property: Property bag of chat thread metadata key-value pairs.
+     *
+     * @param metadata the metadata map to set.
+     * @return the ChatThreadProperties object itself.
```

Comment:
I think we don't need to keep the "Set/Get the <name> property:" at the beginning of each setter/getter Javadoc. Docs should be descriptive but succint. I'd polish the docs for all the public models.
```suggestion
    /**
     * Set the metadata property: Property bag of chat thread metadata key-value pairs.
     *
     * @param metadata the metadata map to set.
     * @return the ChatThreadProperties object itself.
```

Reply:
What has been changed in proposed change?

Reply:
I would apply removing of get/set in comments for the changes I introduced in the PR. I would keep existed ones as it is :)

Reply:
> What has been changed in proposed change?

My bad, I didn't mean to embed a code suggestion.